longestPalindrome.js

```diff
@@ -0,0 +1,27 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/longest-palindrome/
+
+const longestPalindrome = function (s) {
+  const hashMap = {};
+  for (let i = 0; i < s.length; i += 1) {
+    const element = s[i];
+    if (!hashMap[s[i]]) {
+      hashMap[element] = 1;
+    } else {
+      hashMap[element] += 1;
+    }
+  }
+  const evenValues = Object.values(hashMap).filter((value) => value % 2 === 0);
+  const oddValues = Object.values(hashMap).filter((value) => value % 2 !== 0);
+  let sum = 1;
+  if (oddValues.length === 0) {
+    sum = 0;
+  }
```

Comment:
Lets leave a describing comment for **why sum here can take value of either 0 or 1** (depending on odd values being present) because it isn't obvious.